packaging/debian/control

```diff
@@ -0,0 +1,19 @@
+Source: libsxg
+Priority: optional
+Maintainer: Hiroki Kumazaki <kumagi@google.com>
+Build-Depends: debhelper (>= 11)
```

Comment:
I believe this needs a few more packages to build, e.g. `cmake`?

Reply:
Appended.